app/controllers/users_controller.rb

```diff
@@ -6,7 +6,7 @@ def auth_user
   end
 
   def show
-    @user = current_user
+    @user = User.find(params[:id])
```

Comment:
Response from Kate: If it's a quick fix it would be better if you could only see your group but I don't think it's a show-stopper. Let me put it this way -- if it would take <1 hour to have it be JUST peer groups let's change it. If not, let's leave it as is.

If we leave it as is, let's remove email from the show page.